x-pack/plugins/secops/server/lib/events/query.dsl.ts

```diff
@@ -60,7 +60,7 @@ export const buildQuery = (options: RequestOptions) => {
     if (sortField.sortFieldId) {
       const field: string =
         sortField.sortFieldId === 'timestamp' ? '@timestamp' : sortField.sortFieldId;
-      const dir: SortRequestDirection = sortField.direction === 'descending' ? 'desc' : 'asc';
+      const dir: SortRequestDirection = sortField.direction || 'desc';
```

Comment:
Do you want the null fallback value here to be `desc`? Previously if `sortField.direction === null` the fallback value would be `asc`.

Reply:
yes I realized that I thought it will be better to be 'desc'

Reply:
let me fix that differently

Reply:
there is no need to be optional